modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/StructuredContentDTOConverter.java

```diff
@@ -220,15 +218,17 @@ public StructuredContent toDTO(
 	private Map<Locale, String> _filterDescriptionMap(
 		Map<Locale, String> descriptionMap) {
 
-		Set<Map.Entry<Locale, String>> set = descriptionMap.entrySet();
+		Map<Locale, String> filterDescriptions = new HashMap<>();
```

Comment:
@JanisZhang See https://liferay.atlassian.net/wiki/spaces/ENGR/pages/1511752519/Java+conventions+in+Liferay#%5BinlineExtension%5DMaps-are-plurals%2C-with-suffix for naming.